### ISSUE TITLE
Improve documentation re 'ipurge' and friends?

### ISSUE THREAD

**Issue by wdpartain** [cyrusimap/cyrus-imapd#3001]
The documentation (man pages) re "deleting" email could use clarification.

Consider the task "Delete very old email" (on an ad hoc basis, say).  For this, we have 'ipurge', which "deletes" email, and which (happily) has a "dry run" flag.  You run 'ipurge -n' to see what might happen, then 'ipurge' to do it for real.  Great!

It will come as something of a surprise that days/years later all that "deleted" email is still on your disk(s), assuming the (not too visible) defaults for expunging and deleting, i.e. "delayed".

Could you add something to the 'ipurge' man page, along the lines of "You need to run cyr_expire -X ..." in order for the "deleted" email to really go away?

Related: for 'cyr_expire -X <n>', are those <n> "expunge-days" something to do with the age of the messages, or how long it's been since I did an 'ipurge'?  (It seems to be the latter.)  If I do an 'ipurge -d 1800' and then a cyr_expire -X 1799', I believe the messages will remain on the disk(s).

While we're here: a "dry run" flag for 'cyr_expire' would be welcome.  Getting a response 'expunged 123,456,789 messages' is something you'd rather not have come as a surprise :-)

Also: since "mailbox annotations" seem to play a role in these processes, could the "See Also" point to some way to see what these annotations are?

**Comment by rjbs** [comment]
I agree, this should be improved, but this issue won't be needed to get there.  Sorry you waited so long to hear that. 😉
